src/martialarts.h

```diff
@@ -209,11 +209,17 @@ class martialart
         bool has_weapon( const std::string &item ) const;
         // Is this weapon OK with this art?
         bool weapon_valid( const item &u ) const;
+        // Getter for player style change message
+        std::string get_initiate_player_message() const;
+        // Getter for NPC style change message
+        std::string get_initiate_npc_message() const;
+
 
         matype_id id;
         bool was_loaded = false;
         std::string name;
         std::string description;
+		std::vector<std::string> initiate;
```

Comment:
Indention is off, maybe a missing space or maybe a tab.

Reply:
Yeah, turns out CodeBlocks wasn't doing the tabs to spaces conversion correctly. I'll be fixing that right now.